### PR TITLE
Cthalmann/navigation

### PR DESCRIPTION
**Pull request by cthalmann**
## Changes in this PR.
- [X] Adds basic top navigation bar for streamwebs
- [ ] CSS

## Testing this PR.
1.     Start the docker container with docker-compose up
2.     Visit http://localhost:8000/streamwebs

### Expected Output.
Please click on the pictures to zoom in.

When logged out:
loggedout

When logged in:
loggedin

@osuosl/devs

### CODE REVIEW

File: streamwebs_frontend/streamwebs/views.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 # Create your views here.
 def index(request):
+    request.current_app = 'streamwebs'
```
Comment (Kennric) [inline streamwebs_frontend/streamwebs/views.py]: Are we certain we need this? Can you remove and test?

File: streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<div id="topbar">
+    <div class="navbar">
+        <div class="navbar right"
+             align="right"></div>
```
Comment (Kennric) [inline streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html]: You probably don't need to/shouldn't break this into two lines - html doesn't follow Python style rules and newlines inside a tag is not great, especially for such short tags.

Comment (elenatorrey) [inline streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html]: I am wondering if you should remove the alignment because Alex might try to style it in css and there might be some conflicts... you should probably ask @subnomo

**Comment by elenatorrey** [comment]
Good job on navigation. I think we should add another list item to the navigation to send users to the home page. I know resources bring you to the index page but I assume it is just temporary thing since we don't have resources yet. But we do need a home page.

File: streamwebs_frontend/streamwebs/templatetags/base_extras.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+from django.core.urlresolvers import reverse
+
+register = template.Library()
+
+@register.simple_tag
+def navactive(request, urls):
+    if request.path in ( reverse(url) for url in urls.split() ):
+        return "active"
+    return ""
+"""
```
Comment (alxngyn) [inline streamwebs_frontend/streamwebs/templatetags/base_extras.py]: Is this file needed since everything is commented out?

File: streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        <a href="{% url 'streamwebs:index' %}">Resources</a>
+        {% if user.is_authenticated %}
+            <a href="{% url 'streamwebs:logout' %}">Logout</a>
+        {% else %}
+            <a href="{% url 'streamwebs:login' %}">Login</a>
+            <a href="{% url 'streamwebs:register' %}">Create Account</a>
+        {% endif %}
+        </div>
+        </nav>
+        <div class="navbar left"></div>
```
Comment (alxngyn) [inline streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html]: lets remove the empty div?

File: streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<div id="topbar">
+    <div class="navbar">
+        <div class="navbar right" align="right"></div>
```
Comment (subnomo) [inline streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html]: We don't need this empty div, do we? It should probably be removed.

File: streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        <div class="navbar content"
+             align="right">
```
Comment (subnomo) [inline streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html]: Like @Kennric said, you needn't split this into two lines.

File: streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+        <a href="{% url 'streamwebs:sites' %}">Sites</a>
+        <a href="{% url 'streamwebs:index' %}">Resources</a>
+        {% if user.is_authenticated %}
+            <a href="{% url 'streamwebs:logout' %}">Logout</a>
+        {% else %}
+            <a href="{% url 'streamwebs:login' %}">Login</a>
+            <a href="{% url 'streamwebs:register' %}">Create Account</a>
+        {% endif %}
+        </div>
+        </nav>
```
Comment (subnomo) [inline streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html]: Make sure you indent the inner tags, so we can easily see which elements are children of another element.

**Comment by subnomo** [comment]
Looks good. :+1:

**Comment by Kennric** [comment]
Looks good to me. +1
